### PR TITLE
Add image-lightbox

### PR DESCRIPTION
**Pull request by kul3r4**
Related to #578.
Carousel is still not supported on a lightbox (ref [here](https://github.com/ampproject/amphtml/issues/8739)) but we could show how it's possible to add an image lightbox for each product.

@ericlindley-g We briefly talked about this, WDYT?
@sebastianbenz PTAL

### CODE REVIEW

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 1]
```diff
@@ -448,13 +452,24 @@ <h2 class="m1">Apple</h2>
...
                 width="1024" height="682"
-                layout="responsive">
+                layout="responsive" on="tap:gallery-lightbox"
+                role="button" tabindex="0">
         </amp-img>
     </amp-carousel>
+    <!-- ## Full screen image -->
+    <!-- The `amp-image-lightbox` component allows the user to expand an image to fill the viewport.
+    By clicking on each image from the product gallery, the user can see the product image by using the full screen.
+    Learn more [here](https://ampbyexample.com/components/amp-image-lightbox/) -->
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product_Page.html]: use relative links to ABE

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 2]
```diff
@@ -448,13 +452,24 @@ <h2 class="m1">Apple</h2>
...
+    <amp-image-lightbox id="gallery-lightbox" layout="nodisplay">
+      <div class="lightbox" on="tap:gallery-lightbox.close" role="button"
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product_Page.html]: This is not visible. Can you show a close button in the top corner, but also support closing by clicking anywhere?

**Comment by ericlindley-g** [comment]
+1 This is great!

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
The image overlaps the button:

screen shot 2017-05-02 at 7 26 58 pm

**Comment by kul3r4** [comment]
Ok, is this mobile or responsive?
Attached is what I get:
product-lightbox

**Comment by sebastianbenz** [comment]
rotate the screen to landscape and you'll notice. In your case the image
simply doesn't cover the button.

**Comment by kul3r4** [comment]
Right, I was able to reproduce it, thanks!
I updated the code using z-index and position absolute, what do you think?

**Comment by sebastianbenz** [comment]
Can you rebase and merge?

**Pull request merged**